### ISSUE TITLE
Is not match the paper about compute Omni-loss with D_fake ?

### ISSUE THREAD

**Issue by Johnson-yue** [PeterouZh/Omni-GAN-PyTorch#6]
As the paper said,

when train D_real_loss by using this code :
https://github.com/PeterouZh/Omni-GAN-PyTorch/blob/564a586fed6ce51ef73933d8815d94ce077c4e5c/exp/omni_loss/train_fns.py#L68

real label is positive and the fake should be negative
but in your code ;
https://github.com/PeterouZh/Omni-GAN-PyTorch/blob/564a586fed6ce51ef73933d8815d94ce077c4e5c/exp/omni_loss/train_fns.py#L79

Is it should be : 
`D_loss_fake = omni_loss(pred=D_fake, positive= None, negative=D_fake_positive, default_label=0)`

**Comment by Johnson-yue** [comment]
sorry  it is my fault
